packages/client/src/frames/Types.js

```diff
@@ -45,10 +47,12 @@ export type DestinationSetupFrame = {|
   type: 0x02,
   majorVersion: ?number,
   minorVersion: ?number,
-  inetAddress: IPv4 | IPv6,
+  inetAddress?: IPv4 | IPv6,
   group: string,
+  connectionId: ConnectionId,
```

Comment:
Just to keep consistent with your serialization order, perhaps move this to its correct position in the type def? Just to avoid future confusion as we discussed on the phone.

Reply:
Good call, I'll make the change.